packages/filesystem/src/browser/file-dialog-service.ts

```diff
@@ -0,0 +1,46 @@
+/********************************************************************************
```

Comment:
opening a file dialog is needed from both `workspace` and `navigator` packages. And I am thinking it could possibly be used in other packages in future. Therefore I am making an injectable service which could be reused.

Reply:
good idea